src/App.tsx

```diff
@@ -14,11 +14,103 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+enum SortBy {
+  None = 'none',
+  Name = 'name',
+  Length = 'length',
+}
+
+const App: React.FC = () => {
+  const [isStarted, setStart] = useState(false);
+  const [isReversed, reverse] = useState(false);
```

Comment:
be consistent in naming conventions
```suggestion
  const [isStarted, setIsStarted] = useState(false);
  const [isReversed, setIsReversed] = useState(false);
```